packages/fusion-runtime/src/utils.ts

```
@@ -246,6 +250,17 @@
         executor,
       );
     }
+    if (tracer?.subgraphExecute) {
+      const originalExecutor = executor;
+      executor = (executionRequest) => {
+        let result: MaybePromise<MaybeAsyncIterable<ExecutionResult<any, any>>>;
+        const traced$ = tracer.subgraphExecute({executionRequest}, () => {
+          result = originalExecutor(executionRequest);
+          return isPromise(result) ? result.then(() => undefined): undefined;
+        })
+        return isPromise(traced$) ? traced$.then(() => result) : result!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)